package.json

```diff
@@ -170,8 +171,11 @@
       "@oclif/plugin-autocomplete",
       "@oclif/plugin-help",
       "@oclif/plugin-not-found",
+      "@oclif/plugin-plugins",
       "@oclif/plugin-warn-if-update-available"
     ],
+    "pluginPrefix": "rdme-plugin",
+    "scope": "readme",
```

Comment:
more info here: https://github.com/oclif/plugin-plugins?tab=readme-ov-file#friendly-names

also i opened up a PR since the docs don't mention `pluginPrefix` but it's definitely a thing: https://github.com/oclif/plugin-plugins/pull/1113